README.md

```diff
@@ -1,23 +1,64 @@
 # Flowshutter
 
-Flowshutter is a customize camera remote written in micropython.
+<p align="center">
+  <a href="https://gyroflow.xyz">Homepage</a> •
+  <a href="https://discord.gg/WfxZZXjpke">Discord</a> •
+  <a href="https://github.com/gyroflow/flowshutter/issues">Report bug</a> •
+  <a href="https://github.com/gyroflow/flowshutter/issues">Request feature</a>
+</p>
 
-It can synchronize vedio recording of the camera and blackbox logging of your motion logger (in most cases betaflight/emuflight controller with flash chip or SD card). 
+<p align="center">
+  <a href="https://github.com/gyroflow/flowshutter/languages/top">
+    <img src="https://img.shields.io/github/languages/top/gyroflow/flowshutter" alt="Languages">
+  </a>
+  <a href="https://github.com/gyroflow/flowshutter/">
+    <img src="https://img.shields.io/github/contributors/gyroflow/flowshutter?color=dark-green" alt="Contributors">
+  </a>
+  <a href="https://github.com/gyroflow/flowshutter/issues/">
+    <img src="https://img.shields.io/github/issues/gyroflow/flowshutter" alt="Issues">
+  </a>
+  <a href="https://github.com/gyroflow/flowshutter/blob/master/LICENSE">
+    <img src="https://img.shields.io/github/license/gyroflow/flowshutter" alt="License">
+  </a>
+</p>
 
-It was designed to be used with gyroflow software to provide you the best open source video stabilization experience.
+Flowshutter is a customize camera remote, designed to be the most flexible and reliable gyroflow camera external motion logger. It can provide precise synchronization of camera vedio recording and motion logger (betaflight/emuflight FC) recording. 
 
-## Hardware
+It was designed to be used with gyroflow software to provide you one of the best open source video stabilization experience.
```

Comment:
It was designed to be used with the Gyroflow software to provide you one of the best open source video stabilization experiences.

Reply:
accepted